### PR TITLE
Add and import enlighted-link

### PR DESCRIPTION
**Pull request by tomalec**
Add [Juicy/enlighted-link](https://github.com/Juicy/enlighted-link)@0.0.1, to allow importing dependencies from shadow roots

Import it in starcounter.html, so any composition could use it.

https://github.com/Starcounter/RebelsLounge/issues/258

### CODE REVIEW

File: src/StarcounterClientFiles/wwwroot/sys/starcounter.html [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <link rel="import" href="palindrom-client/palindrom-client.html">
+<link rel="import" href="enlighted-link/enlighted-link.html">
```
Comment (alshakero) [inline src/StarcounterClientFiles/wwwroot/sys/starcounter.html:2]: Could you make these two use `/sys` urls instead please? That would be good for upcoming ApplicationCache support. Urls need to be the same everywhere and non-`sys` urls work only here.

**Review by alshakero — APPROVED** [review]
LGTM (sorry for the delay)

File: src/StarcounterClientFiles/wwwroot/sys/starcounter.html (hunk 1 above)
Comment (tomalec) [reply src/StarcounterClientFiles/wwwroot/sys/starcounter.html:2]: well... we have plenty of non-`sys` URLs, at `/sys/*/*.html` is that a problem for cache?

Comment (alshakero) [reply src/StarcounterClientFiles/wwwroot/sys/starcounter.html:2]: Good point

**Pull request merged**